internal/utils/config.go

```diff
@@ -99,6 +100,24 @@ func MakeConfig() (*types.Config, error) {
 	ldapUserFilter := getEnv("LDAP_USERFILTER", "(cn=%s)")
 	tenant := strings.ToLower(getEnv("TENANT", KubiTenantUndeterminable))
 
+	podSecurityAdmissionEnforcement, errPodSecurityAdmissionEnforcement := podSecurity.ParseLevel(strings.ToLower(getEnv("PODSECURITYADMISSION_ENFORCEMENT", string(podSecurity.LevelRestricted))))
+
```

Comment:
I would add a comment, for the lazy ppl like me, to say:
```
# No need to state a default or crash, because kubernetes defaults to restricted. 
```